101-event-grid-cloudevents/azuredeploy.json

```diff
@@ -0,0 +1,189 @@
+    }
+  },
+  "resources": [
+    {
+      "comments": "### RESOURCE - LOGIC APP - EVENT HANDLER ###",
+      "apiVersion": "[variables('logicApp').apiVersion]",
+      "type": "Microsoft.Logic/workflows",
+      "name": "[variables('logicApp').name]",
+      "location": "[variables('logicApp').location]",
+      "tags": "[variables('tags')]",
+      "properties": {
+        "state": "Enabled",
+        "definition": {
+          "$schema": "https://schema.management.azure.com/providers/Microsoft.Logic/schemas/2016-06-01/workflowdefinition.json#",
+          "contentVersion": "1.0.0.0",
+          "parameters": {},
+          "triggers": {
+            "manual": {
+              "type": "Request",
+              "kind": "Http",
+              "inputs": {
+                "method": "POST",
+                "schema": {}
+              }
+            }
+          },
+          "actions": {},
+          "outputs": {}
+        }
+      }
+    },
+    {
+      "comments": "### RESOURCE - CUSTOM EVENT GRID TOPIC ###",
+      "apiVersion": "[variables('eventGridTopic').apiVersion]",
+      "type": "Microsoft.EventGrid/topics",
+      "name": "[variables('eventGridTopic').name]",
+      "location": "[variables('eventGridTopic').location]",
+      "tags": "[variables('tags')]",
+      "properties": {
+        "inputSchema": "[variables('eventGridTopic').schema]"
+      }
+    },
+    {
+      "comments": "### RESOURCE - EVENT GRID SUBSCRIPTION ###",
+      "apiVersion": "[variables('eventGridSubscription').apiVersion]",
+      "type": "Microsoft.EventGrid/topics/providers/eventSubscriptions",
+      "name": "[concat(variables('eventGridTopic').name, '/Microsoft.EventGrid/', variables('eventGridSubscription').name)]",
+      "location": "[variables('eventGridSubscription').location]",
+      "tags": "[variables('tags')]",
+      "dependsOn": [
+        "[variables('logicApp').name]",
+        "[variables('eventGridTopic').name]"
+      ],
+      "properties": {
+        "eventDeliverySchema": "[variables('eventGridSubscription').deliverySchema]",
+        "destination": {
+          "endpointType": "[variables('eventGridSubscription').endpoint.type]",
+          "properties": {
+            "endpointUrl": "[listCallbackUrl(variables('logicApp').triggerId, variables('logicApp').apiVersion).value]"
+          }
+        },
+        "filter": {
+          "includedEventTypes": "[variables('eventGridSubscription').filter.includedEventTypes]",
+          "isSubjectCaseSensitive": "[variables('eventGridSubscription').filter.isSubjectCaseSensitive]"
+        }
+      }
+    }
+  ],
+  "outputs": {
+    "eventGridTopicEndpointUrl": {
+      "type": "string",
+      "value": "[reference(variables('eventGridTopic').name).endpoint]"
+    },
+    "eventGridTopicSasKey": {
+      "type": "string",
+      "value": "[listKeys(variables('eventGridTopic').resourceId, variables('eventGridTopic').apiVersion).key1]"
```

Comment:
use a literal apiVersion

Reply:
Again, the API version is repeating. Therefore, I intentionally extracted to the variables section. Is it an issue?